### PR TITLE
task 101 completed

### PR DESCRIPTION
**Pull request by Sanaiqbal-dev**
Task 101:Homepage
User Story: https://dev.azure.com/CodeNinjaConsulting/Talent%20Pool/_workitems/edit/1428

### CODE REVIEW

File: portfolio-project-react/src/components/Content.jsx
Comment (aaminbaig) [inline portfolio-project-react/src/components/Content.jsx:1]: Every component should have a folder of its own.

Its component folder should contain the jsx file, css file and so on ... the content that is scoped only for that specific component.

File: portfolio-project-react/src/components/Details.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const Details = () => {
+  return (
+    <div className='details-section'>
+      <h1 class='title-about'>About</h1>
+      <p id='intro-data'>
+        I am a highly experienced Full Stack Web Developer with a proven track
```
Comment (aaminbaig) [inline portfolio-project-react/src/components/Details.jsx]: All of your static content, that includes strings and meaningful numeric values should be inside a newly created CONSTANTS file. 

Each CONSTANTS file should be limited to its component, if the the static content is being used other than the component then it should be placed in the root CONSTANTS file.

File: portfolio-project-react/src/components/Details.jsx
Comment (aaminbaig) [inline portfolio-project-react/src/components/Details.jsx:1]: I'm unable to locate their CSS files, they should be separately maintained.

File: portfolio-project-react/src/components/Picture.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+const Picture = ({ url, size }) => {
```
Comment (aaminbaig) [inline portfolio-project-react/src/components/Picture.jsx]: what if i want a separate height and width of the image?

**Review by aaminbaig — CHANGES_REQUESTED** [review]

File: portfolio-project-react/src/components/Picture.jsx (hunk 2 above)
Comment (Sanaiqbal-dev) [reply portfolio-project-react/src/components/Picture.jsx]: For that I can pass seperate props for width and height, and than set image's dimensions accordingly, but requirement mentioned in acceptance criterea was to pass single property i.e. size to specify both width and height of image.

File: portfolio-project-react/src/components/Details/Details.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <hr />
+
+      <h3 className={styles.inlineDiv}>Contact:</h3>
+      <p className={styles.inlineDiv}>{CONTACT}</p>
+      <br />
+      <h3 className={styles.inlineDiv}>Email:</h3>
+      <p className={styles.inlineDiv}>{EMAIL}</p>
+      <div className={styles.details}>
+        <div>
+          <h1>Work Experience</h1>
```
Comment (aaminbaig) [inline portfolio-project-react/src/components/Details/Details.jsx]: Even the sub-heading text should be in the contants file.

File: portfolio-project-react/src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import styles from "./Footer.module.css"
+const Footer = () => {
+  return (
+    <footer>
+      <a>Linkedin</a>
```
Comment (aaminbaig) [inline portfolio-project-react/src/components/Footer/Footer.jsx]: also in the contants file as its a static text.

File: portfolio-project-react/src/components/Header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import styles from "./Header.module.css"
+
+const Header = () => {
+  return (
+    <header>
+        <a>Login</a>
```
Comment (aaminbaig) [inline portfolio-project-react/src/components/Header/Header.jsx]: static text

**Review by aaminbaig — CHANGES_REQUESTED** [review]

**Review by aaminbaig — APPROVED** [review]

**Pull request merged**